Document alerts and notifications for NOM 1.6 (#106)
neo4j/docs-ops-manager: modules/ROOT/pages/monitoring/metric-manager.adoc

:description: This section describes the Metric manager of Neo4j Ops Manager.

The metrics page offers rich charts and tables with data related to your monitored DBMSs.

> Currently six time windows are supported, from 30 minutes (live) up to 30 days.

Reviewer: Do we specifically need to mention how many time windows we have?

Author: No we don't. I just updated the count :). Do you know if the shortest time window is different from the others (because it is called "live"? Does (only) "live" updates automatically?

Reviewer: No, there is no difference. It just used to be called live for some reason.. Better to remove this reference.

To allow efficient metric storage, metrics data older than 3 days is aggregated to hourly summary.

The table in the *Host Metrics* tab and *Instance Metrics* tab includes static data for our DBMSs, like the CPU cores of the system (physical and logical) and memory.

image::dashboard.png[width=800]

Charts allow for zooming/panning by clicking and specifying the area of interest by dragging the cursor.
To reset zoom in the chart you can press double click inside the chart.
Zoom/reset zoom affects all charts displayed on the page to allow investigating different metrics in a specific time range.

== Setting metric thresholds

To inform administrators about noticeable changes in metrics by showing alerts and sending notifications, NOM supports defining metric thresholds directly from the metrics chart.
This can be accomplished using "Threshold" menu item of the  "..." menu of the metric chart.

> In the example below, a threshold of 6.98GB was added in the chart for "Host memory used" metric.

Reviewer: Might be nicer to have an example with a non-byte unit since that looks a bit nicer. Plan is to improve the size unit input further on.

> image::metric-threshold.png[width=800]

Reviewer: Take new picture with the fixed alignment :)

:description: This section describes the Metric manager of Neo4j Ops Manager.

The metrics page offers rich charts and tables with data related to your monitored DBMSs.
The time window for displayed data can be switched on top left of the metrics page.

[NOTE]
====
To allow efficient metric storage, metrics data older than 3 days is aggregated to hourly summary.
====

> Clicking on "Show Resources" button in the *Host* tab and *Instance* tab shows a table with static data for managed DBMSs, like the number CPU cores of the system (physical and logical) and memory/disk capacities.

Reviewer 2: ```suggestion
The "Show Resources" button in the *Host* tab and *Instance* tab shows a table with static data for managed DBMSs, like the number CPU cores of the system (physical and logical) and memory/disk capacities.
```

image::dashboard.png[width=800]

Charts allow for zooming/panning by clicking and specifying the area of interest by dragging the cursor.

> To reset zoom in the chart you can press double click inside the chart.

Reviewer 2: ```suggestion
To reset zoom in the chart you can double click inside the chart.
```
I assume this is what it should be. You can't "press double click", either you press something or you double-click something.

Zoom/reset zoom affects all charts displayed on the page to allow investigating different metrics in a specific time range.

== Setting metric thresholds

To inform administrators about noticeable changes in metrics by showing alerts and sending notifications, NOM supports defining metric thresholds directly from the metrics chart.
This can be accomplished using "Threshold" menu item of the  "..." menu of the metric chart.

In the example below, a threshold of 80% was added in the chart for "Host CPU Usage" metric.

image::metric-threshold.png[width=800]

After confirming the threshold dialog:

> * The corresponding Condition, which will be used to trigger Alerts and Notifications, will be created and can be managed using the Conditions tab (see xref:./alerts-conditions.adoc#_managing_conditions[Managing Conditions]).

Reviewer 2: ```suggestion
* The corresponding Condition, used to trigger Alerts and Notifications, is created and can be managed using the Conditions tab (see xref:./alerts-conditions.adoc#_managing_conditions[Managing Conditions]).
```

> * If the metric value will exceed the threshold during the defined grace period (5 minutes), the Condition will evaluate to `true`, causing NOM to trigger an Alert and (if configured), send Notifications (see xref:./notifications.adoc[Notifications]).

Reviewer 2: ```suggestion
* If the metric value exceeds the threshold during the defined grace period (5 minutes), the Condition evaluates to `true`, causing NOM to trigger an Alert and (if configured), send Notifications (see xref:./notifications.adoc[Notifications]).
```

> * The metric chart will display the defined threshold in the chart as a dashed red line.

Reviewer 2: ```suggestion
* The metric chart displays the defined threshold in the chart as a dashed red line.
```

> Disabling the threshold can be accomplished by opening the Threshold dialog again and unchecking the "Enable threshold" checkbox.

Reviewer 2: ```suggestion
To disable the threshold, open the threshold dialog and uncheck the "Enable threshold" checkbox.
```

> Doing so will remove this Condition from the system.

Reviewer 2: ```suggestion
This removes this Condition from the system.
```
